pkg/converter/profileConverter.go

```diff
@@ -0,0 +1,33 @@
+package converter
+
+import (
+	"github.com/projectcalico/libcalico-go/lib/api/unversioned"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"k8s.io/client-go/pkg/api/v1"
+
+)
+const NS_POLICY_ANNOTATION = "net.beta.kubernetes.io/network-policy"
```

Comment:
Note we just updated the Python code so that it no longer looks at this annotation - the new NetworkPolicy spec doesn't respect that any more, so the controller no longer needs to look for it.